index.js

```diff
@@ -37,19 +38,32 @@ module.exports.iMeanMost =
 module.exports.butWhatActuallyIsCinnamonToastCrunchThough =
   function nobodyEvahExplainedItToMe() {
     return "Scientists still aren't sure, but we have a suspicion that it " +
-      "just might be a cereal involving cinnamon, toast, and something " +
-      "crunchy!"
+      "just might be something edible involving cinnamon, toast, and something " +
```

Comment:
@towerofnix, The word "Cereal" isn't in the name. So to make the sentence more fun, let's just say what the name says.

Reply:
W-what does this have to do with having it with milk???